### PR TITLE
Advanced-settings

### PR DESCRIPTION
**Pull request by tibetsprague**

### CODE REVIEW

File: src/views/Settings.vue [hunk 1]
```diff
@@ -394,39 +410,39 @@
 </template>
 
 <script lang="ts">
-import { defineComponent, PropType } from "vue";
```
Comment (tibetsprague) [inline src/views/Settings.vue:397]: I reordered and cleaned this up a bunch. I like to order things with 3rd party modules first, then our stuff, in alphabetical order

Comment (matthme) [reply src/views/Settings.vue:397]: great!

File: src/views/Settings.vue [hunk 2]
```diff
@@ -122,54 +124,68 @@
...
-            @click="openPublishAppDialog"
-            style="height: 36px; border-radius: 8px; padding: 0 20px; margin-top: 10px;"
-            >{{ $t("settings.publishAnApp") }}
-          </HCButton>
+        <div class="row section-container">
+          <div class="column" style="flex: 1;">
+            <div style="font-size: 18px">{{ $t("settings.launcherConfiguration") }}</div>
+            <span style="font-size: 14px">{{ $t("settings.launcherConfigurationDescription") }}</span>
+          </div>
+          <HCButton style="margin: 4px 6px;" @click="($refs.configDialog as typeof Config).open()">Configure Launcher</HCButton>
```
Comment (matthme) [inline src/views/Settings.vue:184]: This line here throws a type-check error in my IDE although it compiles. Github's check also throws an error. But I think I will still merge it and maybe move the dialog opening logic to a method to be able to use types.

**Review by matthme — APPROVED** [review]
Thank you, looks great :)

File: src/views/Settings.vue (hunk 2 above)
Comment (tibetsprague) [reply src/views/Settings.vue:184]: weird, I dont know why its doing that. I did create an open method

**Pull request merged**